languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -34110,6 +34110,169 @@ though, and has since been slightly modified)-->
+                  <token regexp="yes">lui|elle|eux|nous|vous|elles</token>
+                  <token regexp="yes">seule?s?</token>
+                  <token>,</token>
+                  <token postag="V.*" postag_regexp="yes"/>
+                  <token regexp="yes">[^,‘’–)—."\/;\]\|:(\…'«»“”-]+</token>
+                  <token>,</token>
+              </antipattern>
+              <antipattern>
+                  <token postag="SENT_START"/>
+                  <token regexp="yes">lui|elle|eux|nous|vous|elles</token>
+                  <token regexp="yes">seule?s?</token>
+                  <token>,</token>
+                  <token postag="V.*" postag_regexp="yes"/>
+                  <token>-t-il</token>
+              </antipattern>
+              <pattern>
+                  <token postag="SENT_START"/>
+                  <token regexp="yes">lui|elle|eux|nous|vous|elles</token>
+                  <marker>
+                      <token regexp="yes">seule?s?</token>
+                      <token>,</token>
+                      <token  min="0" postag="R pers obj.*" postag_regexp="yes"/>
+                      <token postag="V.*" postag_regexp="yes"/>
+                  </marker>
+              </pattern>
+              <message>La virgule doit être omise dans ce contexte.</message>
+              <suggestion>\3 \5 \6</suggestion>
+              <example correction="seul me connait">Lui <marker>seul, me connait</marker>.</example>
+          </rule>
+          <rule>
+              <antipattern>
+                  <token>ou</token>
+                  <token>autrement</token>
+                  <token>dit</token>
+                  <token>,</token>
+                  <token postag="D.*" postag_regexp="yes"/>
+              </antipattern>
+              <antipattern>
+                  <token postag="SENT_START"/>
+                  <token>autrement</token>
+                  <token>dit</token>
+                  <token>,</token>
+                  <token postag="D.*" postag_regexp="yes"/>
+              </antipattern>
+              <antipattern>
+                  <token postag="UNKNOWN"/>
+                  <token>autrement</token>
+                  <token>dit</token>
+                  <token>,</token>
+                  <token postag="D.*" postag_regexp="yes"/>
+              </antipattern>
+              <pattern>
+                  <marker>
+                      <token regexp="yes">(?-i)autrement</token>
+                      <token>dit</token>
+                      <token>,</token>
+                      <token postag="D.*" postag_regexp="yes"/>
+                  </marker>
+              </pattern>
+              <message>La virgule doit être omise dans ce contexte.</message>
+              <suggestion>\1 \2 \4</suggestion>
+              <example correction="autrement dit son">Sa maison <marker>autrement dit, son</marker> lieu de résidence,  ne se trouve pas en France.</example>
+          </rule>
+          <rule>
```

Comment:
I think this is a problem. Didn't you write a rule that puts the comma before words like "car"?
This will lead to a rule-loop (the user can't choose a correction that won't cause an alarm). Please check and make sure.

Reply:
Indeed, this one is a little tricky. I think it can be taken off for now.